### DISCUSSION TITLE
Uninstall instructions might be nice

### DISCUSSION THREAD

**Discussion by haraldreingruber-dedalus** [remotemobprogramming/mob#217] [Ideas]
Hi,

we recently used mob.sh in our mob-programmig meetup and were very happy about how it works.
But we noticed, for some people it's a little barrier installing a tool they don't know just for the sake of a few hours session.

Maybe some uninstall instructions could help in this case, so they see how easy it is to remove it afterwards (not that I think many would).

Could it be that just the `/usr/local/bin/mob` binary has to be deleted?

**Comment by simonharrer** [comment]
Depends on the way you install it. But in the normal case, just delete the binary from the folder. Feel free to create a PR.
